### PR TITLE
TA-1_Branching_Strategy_Programming

### PR DESCRIPTION
**Pull request by GalburG**
Described the Git Workflow in steps - practice from lesson 7 , Git  and Java practice in IntelliJ nr 2

Pls also see my last comment in the code.

### CODE REVIEW

**Review by AnastasiaNeghina — APPROVED** [review]

File: src/main/java/com/galbur/Main.java [hunk 1]
```diff
@@ -2,6 +2,61 @@
...
+        System.out.println(step12);
+        System.out.println(step13);
+        System.out.println(step14);
+        System.out.println(step15);
+        System.out.println(step16);
+        System.out.println(step17);
+        System.out.println(step18);
+
+        // end
+        // For those who gonna review this code pls one of you make any comment on it, me to be able to practice also commit amend action.
```
Comment (AnastasiaNeghina) [inline src/main/java/com/galbur/Main.java:54]: comment )

**Review by AnastasiaNeghina — CHANGES_REQUESTED** [review]
your code looks ok )

**Comment by GalburG** [comment]
Modification done to the code, and commit amend applied

**Review by AnastasiaNeghina — APPROVED** [review]

**Comment by Nadiafol** [comment]
looks good! :)

**Comment by GalburG** [comment]
> looks good! :)

**Review by Nadiafol — APPROVED** [review]
reviewed!

**Pull request merged**